src/laundryScraper.py

```diff
@@ -1,76 +1,152 @@
+"""Iterate through buildingList and machineDict
+   Convert to dynamo format and write info to table
+"""
+def writeDynamo():
+	"""Client with access keys to write to Dynamo tables online"""
+	dynamodb = boto3.client('dynamodb',
+		  aws_access_key_id = AWS_ACCESS_KEY_ID,
+		  aws_secret_access_key = AWS_SECRET_ACCESS_KEY,
+		  region_name = 'us-east-2')
+	"""Go through each building, convert it to dictionary, and put in relevant info in    
+	   Dynamo format
+	   Go through the machines for each of those buildings, do the same
+	   Then, write buildingTable and machineTable to Dynamo table
 	"""
-	for row in table:
-		if row.find('div', 'desc'):
-			machineNum = row.find('div','desc').get_text().strip()
-			
-		if row.find(alt={'inuse','available'}):
-			machineType = row.find(alt={'inuse', 'available'})
-			if str(machineType).find('washer') != -1:
-				machineType = 'Washer'
-			else:
-				machineType = 'Dryer'
-				
-		if row.find('div', 'runtime'):
-			runTime = row.find('div', 'runtime').get_text().strip()
-			if runTime.find('remaining') != -1:
-				runTimeInt = [int(s) for s in runTime.split() if s.isdigit()][0]
-			else:
-				runTimeInt = 0
-				
-			machineId = machineType + '-' + machineNum		
-			if machineId not in foundMachines:
-				foundMachines.add(machineId)
-				machineDict[buildingId].append((Machine(runTime, machineId, buildingId, machineType, runTimeInt, True)))
+	for building in buildingList:
+		
+		buildingTable = dict()
+		machineTable = dict()
+		
+		buildingTable['Buildings'] = list()
+		machineTable['Machines'] = list()
+		
+		newBuilding = dict()
+		newBuilding['PutRequest'] = dict()
+		newBuilding['PutRequest']['Item'] = dictToItem(vars(building))
+		buildingTable['Buildings'].append(newBuilding)
+		
+		for machine in machineDict[building.buildingId]:
+			newMachine = dict()
+			newMachine['PutRequest'] = dict()
+			newMachine['PutRequest']['Item'] = dictToItem(vars(machine))
+			machineTable['Machines'].append(newMachine)
 	
-	"""Using data found above, create Building object
-	   Store in buildingList
-	"""
-	numWashers = sum(p.machineType == 'Washer' for p in machineDict[buildingId])
-	numDryers = sum(p.machineType == 'Dryer' for p in machineDict[buildingId])
-	buildingList.append(Building(buildingId, buildingInfo['friendlyNames'], numWashers, numDryers))
+		buildingResponse = dynamodb.batch_write_item(RequestItems=buildingTable)
+		machineResponse = dynamodb.batch_write_item(RequestItems=machineTable)
+
+if __name__ == '__main__':
+	
+	"""Used to store Building and Machine objects with all the relevant info"""
+	buildingList = list()
+	machineDict = dict()
 
-"""Test"""
-for building in buildingList:
-	print('\n{} #Washers={} #Dryers={}'.format(building.buildingId, building.numWashers, building.numDryers))
-	for machine in machineDict[building.buildingId]:
-		print('{}: {}'.format(machine.machineId, machine.status))
+	scrapeLaundry()
+	writeDynamo()
```

Comment:
Instead of calling these sequentially down here, you should just call `scrapeLaundry` with any arguments (let's say `scrapeLaundry(None, None)`), then at the end of `scrapeLaundry()` call `writeDynamo(buildingList, machineDict)`